Whisky/Views/Setup Views/GPTKInstallView.swift

```diff
@@ -16,6 +16,7 @@ struct GPTKInstallView: View {
         VStack {
             Text("setup.gptk")
                 .font(.title)
+                .fontWeight(.bold)
```

Comment:
Why these changes? The same goes for all subsequent Setup Views adjustments.

Reply:
Bold for titles are more modern in new macos.

Reply:
Okay tried it on my machine and it looks okay. The extra padding isn't necessary tho imo

Reply:
> Okay tried it on my machine and it looks okay. The extra padding isn't necessary tho imo

Will remove then.